Modules/FindCapistrano.cmake

```diff
@@ -1,61 +1,89 @@
-# Find the Capistrano iSDK
+# Find the Capistrano cSDK
 # This module defines
 # CAPISTRANO_FOUND - Capistrano SDK has been found on this system
 # CAPISTRANO_INCLUDE_DIR - where to find the header files
 # CAPISTRANO_LIBRARY_DIR - libraries to be linked
 # CAPISTRANO_BINARY_DIR - common shared libraries to be installed
-# CAPISTRANO_WIN32_BINARY_DIR - 32-bit shared libraries to be installed
-# CAPISTRANO_WIN64_BINARY_DIR - 64-bit shared libraries to be installed
+# CAPISTRANO_LIBRARY_USBPROBE_NAME - USBprobe filename in the lib dir
+# CAPISTRANO_BINARY_USBPROBE_NAME - USBprobe filename in the bin dir
+# CAPISTRANO_LIBRARY_BMODE_NAME - Bmode filename in the lib dir
+# CAPISTRANO_BINARY_BMODE_NAME - Bmode filename in the bin dir
 
-SET( Capistrano_PATH_HINTS
-  "c:/cSDK2013"
-  )
+SET(Capistrano_SDK_VERSION "cSDK2018" CACHE STRING "Which cSDK version to build Plus against")

```

Comment:
Typically we keep these options in the PlusBuild CMakeLists.txt, but this makes a lot more sense. @lassoan thoughts?

Reply:
We prefer to have all SDKs (stored in PLTools repository) that we are building against so that we can include them in automatic testing. However, if that would require any obligations (signing NDA, etc.) then we would rather just exclude the SDK from automatic builds and then maintainers of that specific interface need to do their testing internally.

Reply:
Sorry, in this case I meant specifically having the variable for SDK version inside the Find module as opposed to the PlusBuild CMakeList.txt.

Reply:
Yes, it would be nicer to be able to find a particular SDK version rather than hardcoding it in the Find...cmake script. However, if the script can only work with one particular SDK version then it is OK to hardcode it.

Reply:
This split is related to library names and available functions. And it could be called pre2018/post2018. For possible future versions cSDK2018 should work, if it is header/library name compatible. But the split into versions 2013, 2016 and 2018 is for style and lower cognitive load.